### PR TITLE
Support syzhub for syzkaller

### PR DESCRIPTION
**Pull request by mspectorgoogle**

### CODE REVIEW

File: src/python/bot/fuzzers/syzkaller/config.py [hunk 1]
```diff
@@ -52,6 +53,14 @@ def generate(serial,
   data['procs'] = 1
   data['cover'] = kcov
 
+  syshub_address = environment.get_value('SYZHUB_ADDRESS')
+  syshub_client = environment.get_value('SYZHUB_CLIENT')
+  syshub_key = environment.get_value('SYZHUB_KEY')
+  if syshub_address and syshub_client and syshub_key:
+    data["hub_addr"] = syshub_address
```
Comment (oliverchang) [inline src/python/bot/fuzzers/syzkaller/config.py:60]: nit: single quotes here.

File: src/python/bot/fuzzers/syzkaller/config.py [hunk 2]
```diff
@@ -52,6 +53,14 @@ def generate(serial,
   data['procs'] = 1
   data['cover'] = kcov
 
+  syshub_address = environment.get_value('SYZHUB_ADDRESS')
```
Comment (oliverchang) [inline src/python/bot/fuzzers/syzkaller/config.py:56]: nit: seems more consistent to do this in syzkaller.runner.get_config instead, and and pass them as arguments to this function. Better to keep dependencies on global/env vars in one place.

File: src/python/bot/fuzzers/syzkaller/config.py [hunk 3]
```diff
@@ -23,16 +23,23 @@ def generate(serial,
              vmlinux_path,
              config_path,
              kcov=True,
-             reproduce=True):
+             reproduce=True,
+             syshub_address=None,
```
Comment (inferno-chromium) [inline src/python/bot/fuzzers/syzkaller/config.py:27]: typo syzhub instead of syshub in all variables ?

**Pull request merged**